### ISSUE TITLE
Book 05: Sentence 066

### ISSUE THREAD

**Issue by ftyers** [ftyers/UD_Classical_Nahuatl-FloCo#36]
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/8d9a64dd-cd1d-4bef-88a8-c39f067d0885)

```
# sent_id = Book_05_-_The_Omens.txt:66
# text = Auh intla can ipa otlatvic, in çan ie vncan pilcac, ic moiollalia: amo cenca qujueimati, atle ipan conjtta.
# text[norm] = Auh intla zan ipan otlathuic, in zan ye oncan pilcac, ic moyollalia: amo cenca quihueyimati, atle ipan conitta.
# text[orig] = A·uh·intlacan·ipa¶otlatvic·,·in·çan·ie·vncan·pilcac·,·ic¶moiollalia·:·amo·cenca·qujueimati·,·atle·ipan·conjtta·.
# text[osp] = [...] y si poruentura, quando la yua ver otro dia, le hallaua alli: entonce consolaua se, teniendo por cierto que no era aguero.
# references = A&D/173:1
# tagged = 100.00%
# lemmatised = 100.00%
# analysed = 100.00%
# heads = 100.00%
# relations = 26.09%
1	Auh	auh	CCONJ	_	_	14	cc	_	_
2	intla	tla	SCONJ	_	_	5	mark	_	_
3	can	zan	ADV	_	_	5	advmod	_	_
4	ipa	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	5	obl	_	_
5	otlatvic	tlathuic	VERB	_	Aspect=Prior|Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Pres|VerbForm=Fin	14	advcl	_	_
6	,	,	PUNCT	_	_	11	punct	_	_
7	in	in	SCONJ	_	_	11	mark	_	_
8	çan	zan	ADV	_	_	11	advmod	_	_
9	ie	ye	ADV	_	_	11	advmod	_	_
10	vncan	oncan	ADV	_	_	11	advmod	_	_
11	pilcac	pilca	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Subcat=Intr|Tense=Past|VerbForm=Fin	14	advcl	_	_
12	,	,	PUNCT	_	_	14	punct	_	_
13	ic	ic	ADV	_	_	14	advmod	_	_
14	moiollalia	yollalia	VERB	_	Number[subj]=Plur,Sing|Person[subj]=3|Reflex[obj]=Yes|Subcat=Tran|Tense=Pres|VerbForm=Fin	0	root	_	_
15	:	:	PUNCT	_	_	18	punct	_	_
16	amo	amo	ADV	_	_	18	_	_	_
17	cenca	cenca	ADV	_	_	18	_	_	_
18	qujueimati	hueyimati	VERB	_	Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin	14	_	_	_
19	,	,	PUNCT	_	_	22	punct	_	_
20	atle	atle	PRON	_	_	21	_	_	_
21	ipan	pan	NOUN	_	NounType=Relat|Number[psor]=Sing|Person[psor]=3|State=Pos	22	_	_	_
22	conjtta	itta	VERB	_	Movement=And|Number[obj]=Sing|Number[subj]=Plur,Sing|Person[obj]=3|Person[subj]=3|Subcat=Tran|Tense=Pres|VerbForm=Fin18	14	_	_	_
23	.	.	PUNCT	_	_	14	punct	_	_
```

**Comment by ftyers** [comment]
López Austin:
> Pero si al amanecer ahí estaba colgado, se ponía muy tranquilo su corazón, no estimaba en nada lo que veía.

**Comment by Lguyogiro** [comment]
Here's a slightly different possibility:

![image](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/7957321/3aa00353-b78e-4f44-bd17-a871e0358823)

I'm not super adamant about splitting intla, but it is something we could do. The motivation for `conj(otlathuic, pilcac)` is mostly semantic I guess, since (at least how I understand this sentence) both of these clauses are necessary for them to have a tranquil heart.

the last part _atle ipan conitta_: I think maybe this is like "they saw nothing at/on it" i.e. "no le veían nada". In this case, I think "nothing" is the obj, with the reln expressing "at it."

**Comment by ftyers** [comment]
A&D have:
> But if it dawned and it hung at this very same place, they were consoled thereby. Little did they think of it; as nothing did they esteem it.

- Wouldn't *tla* be `ADV` here in that case?  Or can *tla* be `SCONJ` without *in* too?
- I like the `obj(conitta, atle)`, that makes sense.
- I'm not sure if `conj` is right there, but thinking about it, I'm not sure that my solution is right either...
  - Do we think its head is *otlahuic* or *moyollalia*:
    - *Se ponía muy tranquilo su corazón que estaba ahí colgado*
    - *Si le amaneció que estaba ahí colgado* 
  - Maybe we want `advcl(otlahuic, pilcac)`?

**Comment by Lguyogiro** [comment]
> Wouldn't tla be ADV here in that case? Or can tla be SCONJ without in too?

_tla_ is SCONJ without it, wherein it is used in optative construction to mean something like _if only_ (Launey 9.5). _in tla_ is used for introducing conditional clauses (34.1). Launey writes them separately and mentions that they are often written together. I separated them just since both can appear as subordinators.


> I'm not sure if conj is right there, but thinking about it, I'm not sure that my solution is right either... 

I read this as the condition being made up of two clauses: _amaneció_ and _estaba ahí colgado_ if both of those are true, s/he becomes tranquil of heart. i.e., I think wrt this portent, if it is hanging there in a different time of day, s/he does not become tranquil of heart. Also, if day breaks and it is not hanging there, s/he does not become tranquil of heart. Thus, they are both advcl to _moyollalia_ (hence `conj`).

If we were to say, e.g. that the clause headed by _tlathuic_ is `advcl` to the clause headed by _pilcac_ or vice-versa, I would expect an _ihcuac_ in one of those clauses, indicating that one thing happened when the other did or something like that.

**Comment by ftyers** [comment]
That sounds true, but we need to have a (morpho)syntactic definition of conjunction... So far we have:
1. If there is *auh* or another coordinator (like *anozo*, maybe)
2. If the phrases fulfill a single function (e.g. a coordinated subject) that cannot be duplicated, as in #34.

Also, you could have two different `advcl` that aren't `conj`,
- If it dawned, as it was hanging there, they were content.
- As it was hanging there, if it dawned, they were content.
- If it dawned and as it was hanging there, they were content.

My feeling is that this should be:
![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/533df617-76e6-4f75-a873-862231ac532d)

But am happy to change my mind if we can come up with a good syntactic definition of how to distinguish the coordination here...

**Comment by Lguyogiro** [comment]
> My feeling is that this should be:

shouldn't atle be `obj` instead of `nsubj`?

**Comment by ftyers** [comment]
> > My feeling is that this should be:
> 
> shouldn't atle be `obj` instead of `nsubj`?

Yes

**Comment by Lguyogiro** [comment]
> But am happy to change my mind if we can come up with a good syntactic definition of how to distinguish the coordination here...

I get that we prefer syntactic criteria for determining relations, but how what is the evidence that these are two adverbial clauses modifying _molollalia_? If there are two analyses, neither of which have strong syntactic evidence for them, shouldn't the burden of proof be on the analysis that is inconsistent with the semantics? 


>     If it dawned, as it was hanging there, they were content.
>     As it was hanging there, if it dawned, they were content.
>     If it dawned and as it was hanging there, they were content.

Isn't the last of these examples coordination? Also, these examples speak to my point above about the lack of _ihcuac_.

**Comment by ftyers** [comment]
> I get that we prefer syntactic criteria for determining relations, but how what is the evidence that these are two adverbial clauses modifying molollalia? If there are two analyses, neither of which have strong syntactic evidence for them, shouldn't the burden of proof be on the analysis that is inconsistent with the semantics?

So the definition is, if we have two equally acceptable syntactic analyses, but one is more coherent semantically, choose that one?

> Isn't the last of these examples coordination? Also, these examples speak to my point above about the lack of ihcuac.

Yes, but my point is that semantically they are much of a muchness, but syntactically they are different.

**Comment by ftyers** [comment]
Ok, coming back, how does this look?

![imatge](https://github.com/ftyers/UD_Classical_Nahuatl-FloCo/assets/449545/4f82f589-ffac-405c-af87-8f3d07d2a4d5)

**Comment by Lguyogiro** [comment]
LGTM :+1:

**Comment by ftyers** [comment]
Merged in aba5974.
